jdaviz/app.py

```diff
@@ -148,6 +148,10 @@ def __init__(self, configuration=None, *args, **kwargs):
         # Parse the yaml configuration file used to compose the front-end UI
         self.load_configuration(configuration)
 
+        # If true, link data on load. If false, do not link data to speed up
+        # data loading
+        self.auto_link = kwargs.pop('auto_link')
```

Comment:
Default is True right?
```suggestion
        self.auto_link = kwargs.pop('auto_link', True)
```

Reply:
Yes! Good catch